src/reducers/systemIntakeReducer.test.ts

```diff
@@ -379,7 +380,7 @@ describe('The system intake reducer', () => {
       });
     });
 
-    it('appends successfully created note to the reducer', () => {
+    it('prepends successfully created note to the reducer', () => {
```

Comment:
I forgot to change this in the "save note" PR. I made that update here. Prepending the note is a good idea.